addons/main/XEH_postInit.sqf

```diff
@@ -10,7 +10,7 @@ GVAR(CalloutCacheNamespace) = call CBA_fnc_createNamespace;
         };
         GVAR(buildingModelCache) setVariable [_string, true];
     };
-} forEach (("configName _x isKindof 'building'" configClasses (configFile >> "CfgVehicles")) apply {getText (_x >> "model")});
+} forEach (("private _name = configName _x; _name isKindof 'building' || {_name isKindOf 'Rocks_base_F'}" configClasses (configFile >> "CfgVehicles")) apply {getText (_x >> "model")});
```

Comment:
when we check here for Rocks_base_f we also should check that in shouldSpressPosition L37

Reply:
why? the cache can be filled with any model that is supposed to block suppression, why not add more to it? All checks later will be then "does the name space have this key?"

Reply:
That said the building check could also be dropped then

Reply:
also i think the getText (_x >> "model") should be done in the loop it self. without we are looping 2x over all of those entries.

Reply:
done.